packages/x-charts/src/hooks/useRadarSeries.test.tsx

```diff
@@ -0,0 +1,70 @@
+import { renderHook } from '@mui/internal-test-utils';
+import { expect } from 'chai';
+import * as React from 'react';
+import { useRadarSeries, useRadarSeriesContext } from './useRadarSeries';
+import { RadarSeriesType } from '../models';
+import { Unstable_RadarChart as RadarChart } from '../RadarChart';
+
+const mockSeries: RadarSeriesType[] = [
+  {
+    type: 'radar',
+    id: '1',
+    data: [1, 2, 3],
+  },
+  {
+    type: 'radar',
+    id: '2',
+    data: [4, 5, 6],
+  },
+];
+
+const defaultProps = {
+  series: mockSeries,
+  radar: { metrics: ['A', 'B', 'C'] },
+  height: 400,
+  width: 400,
+};
+
+const options: any = {
+  wrapper: ({ children }: { children: React.ReactElement }) => {
+    return <RadarChart {...defaultProps}>{children}</RadarChart>;
+  },
+};
+
+describe('useRadarSeriesContext', () => {
+  it('should return all radar series when no seriesIds are provided', () => {
+    const { result } = renderHook(() => useRadarSeriesContext(), options);
+    expect(result.current?.seriesOrder).to.deep.equal(['1', '2']);
+    expect(Object.keys(result.current?.series ?? {})).to.deep.equal(['1', '2']);
+  });
+});
+
+// eslint-disable-next-line mocha/max-top-level-suites
```

Comment:
I wonder if we should remove this rule altogether? Do you know why we have it enabled?